### ISSUE TITLE
How to set NO CACHE for develop?

### ISSUE THREAD

**Issue by jet10000** [Jasonette/JASONETTE-iOS#122]
Thanks a lot

**Comment by gliechtenstein** [comment]
Normally this is not necessary if you use your own server, but if you use online code/text hosting sites like github or pastebin they actually cache your content **server-side** and you keep getting the same response.

I did try my best to resolve this problem by adding an action called `$flush`. Here's how you use it:

```
{
  "type": "$flush",
  "success": {
    "type": "$network.request",
    "options: {
      ...
    }
  }
}
```

However, this doesn't work consistently because the problem is not that the client is caching the content, it's that the server is caching content. So even if you flush everything before making the request, github for example will keep returning the same content for quite a while even after you update the file. And there's not much we can do about it. I talk more about this here: http://docs.jasonette.com/#b-get-a-json-server

So bottom line, my suggestion is:

1. Use your own JSON server if you can (You can use libraries like https://github.com/typicode/json-server or just set up a rails or express app that returns json)
2. Or just use https://jasonbase.com (which I created just so that people can easily try these out) if you don't want to bother setting up your own server.

Hope this helps, please reopen if this was not what you were asking. Thanks!

**Comment by siphahle** [comment]
how do you cache data
